### PR TITLE
Add code examples

### PR DESCRIPTION
**Pull request by Delt06**

### CODE REVIEW

**Review by Dantara — COMMENTED** [review]
Maybe we will add idea-specific files to `.gitignore`? What do you think?

**Review by Dantara — APPROVED** [review]

File: code_examples/dense_code.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+routine main() is
+    var a: integer;
```
Comment (vadimkerr) [inline code_examples/dense_code.txt:2]: Remove semicolon for consistency

File: code_examples/records.txt [hunk 2]
```diff
@@ -0,0 +1,26 @@
+type vector is record
+    var x: real is 0
+    var y: real is 0
+    var z: real is 0
+end
+
+routine add(v1: vector, v2: vector): vector is
+    var result is vector
```
Comment (vadimkerr) [inline code_examples/records.txt:8]: Shouldn't it be `var result: vector`?

File: code_examples/dense_code.txt (hunk 1 above)
Comment (Delt06) [reply code_examples/dense_code.txt]: Fixed

File: code_examples/records.txt (hunk 2 above)
Comment (Delt06) [reply code_examples/records.txt]: Fixed (in ``main`` too)

**Review by vadimkerr — APPROVED** [review]

**Pull request merged**